src/main/kotlin/calculator/Calculator.kt

```diff
@@ -5,14 +5,17 @@ class Calculator {
     fun calculate(input: String?): Int {
         require(!input.isNullOrBlank()) { "Invalid input" }
         val inputs = input.split(" ")
+        isValid(inputs)
         return processStrings(inputs)
     }
 
     private fun processStrings(inputs: List<String>): Int {
-        val queue: MutableList<String> = mutableListOf()
+        var queue: MutableList<String> = mutableListOf()
         for (i in 0 until inputs.size) {
             addQueue(inputs[i], queue)
```

Comment:
```suggestion
            queue.add(input[i])
```

메서드로 감싸주지 않고, 그대로 표현해도 좋아보입니다~__